DeAceroDLabs/marketplace-frontend
# [DMP-656] Change 'Ver más' behaviour

## What is done

[[DMP-656] Arreglar "ver más"](https://deacero.atlassian.net/browse/DMP-656)

## How is done

1. Remove action of 'Ver más' button to navigate to Search
2. Add action to 'Ver más' button to display complete array instead of first 6 items
3. Change 'Ver más' button to 'Ver menos' button on click
4. Add action to 'Ver menos' button to display only first 6 items instead of the complete array

## How to test

1. Run the project via `npm start`
5. On Home click on 'Ver más' in case there is more than 6 items on that category
6. After the complete array of objects is shown, 'Ver más' should now read as 'Ver menos'
7. Click on 'Ver menos' and the amount of objects shown should be reduced to only 6 along with 'Ver más'

## Evidence

===================================Before Changes=====================================
![screen-capture (1)](https://user-images.githubusercontent.com/116199447/199357572-f8b8651f-1642-41dd-95d1-86b98bb938dc.gif)




===================================After Changes======================================
![screen-capture](https://user-images.githubusercontent.com/116199447/199357251-464140c0-6efd-4985-b436-7e86e0bcad2a.gif)
